msteams-platform/m365-apps/teamsjs-support-m365.md

```diff
@@ -80,17 +80,17 @@ The following table lists host application support for TeamsJS capabilities that
             <th><a href="#app">app</a></th>
             <td>&#x2713;</td>
             <td>&#x2713;</td>
-            <td>&#x2713;</td>
+            <td>&#x2713*;</td>
```

Comment:
asterisk should go after semicolon (rendering as a typo)

Reply:
good catch thanks